_events/2023-0718-2-9-release-meetings.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 
-eventdate: 2023-07-24 11:30:00 -0700
+eventdate: 2023-08-08 11:30:00 -0700
```

Comment:
@krisfreedain Edit bellow says "August 7th - 2PM PDT".  Was it supposed to be:

```
eventdate: 2023-08-07 14:00:00 -0700
```

Or is this date good but the other wrong?

Reply:
@smortex I wrote it that way in case we did any website updates the morning of the 7th. (We usually do on Mondays) -- it would force this to move from the events to the past events